Edits to PEM pr5409 (#5573)
EnterpriseDB/docs: product_docs/docs/pem/8/registering_database_server.mdx

#### Examples

This example registers a server using only the required parameters. 
The user `admin01` is used to connect to the PEM server, and the credentials supplied for the `--server-*` parameters are used to create a connection from the agent to the monitored server.

> The same details are also used to populate the server connection details in the PEM web application.

Reviewer: I'm not sure we've gained any clarity with the tense change here. To me the current wording implies "after the user presses enter, the user `admin01` will be used...". Changing to the present tense makes it less clear whether we are making a statement about what will happen as a consequence of this action, or just making a general statement about `admin01`. For example, you could read this as meaning "`Admin01` is a special user that exists for the purpose of connecting to the PEM Server"

Author: Okay,  I tried to come up with another way but i'm not sure it's an improvement, so I'll revert it.
```suggestion
This example registers a server using only the required parameters. 
The user `admin01` will be used to connect to the PEM server, and the credentials supplied for the `--server-*` parameters will be used to create a connection from the agent to the monitored server.
The same details will also be used to populate the server connection details in the PEM web application.
```

          --server-user prod_admin 
```

This example specifies different parameters to use for the connection from the agent to the monitored server.
The example overrides the fully qualified domain name provided in `--server-addr` with `localhost` instead and overrides the user to use a `local_monitor` user. 

> This might be a user who's permitted to connect only from the same host for example.

Reviewer: Postgres users are not necessarily people, so I'm not sure 'who' is appropriate. Is there a precedent elsewhere?

Author: Oh, interesting. The "which" was wrong in any case, so I'll change it to this:
```suggestion
This might be a user that can connect only from the same host, for example.
```

> This might be a user who is permitted to connect only from the same host for example.

Author: ```suggestion
This might be a user who's permitted to connect only from the same host for example.
```
